distribution/lib/Standard/Base/0.0.0-dev/src/Data/Ordering.enso

```diff
@@ -27,30 +27,37 @@ from project.Data.Boolean import all
     ```
     type Ordered_Comparator T
         is_ordered = True
-        compare : T -> T -> Ordering
+        compare : T -> T -> (Ordering|Nothing)
         hash : T -> Integer
 
     type Unordered_Comparator T
         is_ordered = False
-        equals : T -> T -> Boolean
+        equals : T -> T -> (Boolean|Nothing)
```

Comment:
It would be nice to get rid of `equals` and always just use `compare` as #5771 suggests.

Reply:
Will be done as a separate PR. Tracked in #5818.